plugins/com.google.cloud.tools.eclipse.appengine.login/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,12 @@ Bundle-ActivationPolicy: lazy
 Require-Bundle: com.google.cloud.tools.app.lib;bundle-version="0.1.0",
  com.google.gson,
  com.google.guava;bundle-version="15.0.0",
+ org.eclipse.core.runtime,
```

Comment:
If you can, avoid including `org.eclipse.core.runtime`.  Most of its functionality has been split out to other bundles like `org.eclipse.equinox.common`, `org.eclipse.equinox.preferences`, `org.eclipse.equinox.registry`, and more.  The reason is that it brings in the kitchen sink and does a few things badly but can't be changed to preserve backwards compatibility.